src/cli.rs

```diff
@@ -32,7 +32,7 @@ pub enum CliCommand {
 		block_monitor: bool,
 		/// Use mortal transactions.
 		#[clap(long)]
-		mortal: Option<u32>,
+		mortal: Option<u64>,
```

Comment:
As discussed offline - I would add a bit of more doc here - what exactly does this number mean?

Reply:
It is mentioned especially in subxt docs: https://github.com/paritytech/subxt/blob/3643f2c694ab64ab6e0b27870a0c6d33cdb7c5a6/core/src/config/default_extrinsic_params.rs#L58-L59, for the function used to set tx params. I wouldn't investigate more than this how the number is used in the runtime for now to double check it, testing should uncover if this understanding doesn't stand.